client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -223,6 +224,10 @@ const siteMigration: Flow = {
 				}
 
 				case STEPS.SITE_CREATION_STEP.slug: {
+					if ( 'import' === actionQueryParam ) {
+						return navigate( addQueryArgs( { skipMigration: true }, STEPS.PROCESSING.slug ) );
+					}
+
 					return navigate( addQueryArgs( { from: fromQueryParam }, STEPS.PROCESSING.slug ) );
```

Comment:
It looks like we'd lose the `from` argument using this new approach. Might it be better to do set up the args object and then call navigate?
```suggestion
					const queryArgs = {
						from: fromQueryParam,
						skipMigration: ( 'import' === actionQueryParam ) ? true: undefined,
					};

					return navigate( addQueryArgs( queryArgs, STEPS.PROCESSING.slug ) );
```

Reply:
Ahh yes, that makes more sense!